src/python/fuzzing/strategy.py

```diff
@@ -46,6 +46,8 @@
     name='recommended_dict', probability=0.10, manually_enable=False)
 VALUE_PROFILE_STRATEGY = Strategy(
     name='value_profile', probability=0.33, manually_enable=False)
+PEACH_GRAMMAR_MUTATION_STRATEGY = Strategy(
+    name='peach_grammar_mutation', probability=0.10, manually_enable=True)
```

Comment:
This is fine to start, but we should likely increase it later. Since relatively few jobs will support this, we'll want to run it more often so that it's actually used when it can be used.

Reply:
Ack

Reply:
Thinking more, could you add a TODO here so we don't forget about that? Sorry for not mentioning it last time.

Reply:
Done